### DISCUSSION TITLE
This API doesn't have any public projects. Please contact your administrator.

### DISCUSSION THREAD

**Discussion by Desh-Deepak-Dhobi** [directus/api#2518] [Q&A] [answered]
I have been struggling to solve the issue with directus for a few days. I have not been able to get a solution for it. The issue is that when I hit the domain it shows the following error page stating that"This API doesn't have any public projects. Please contact your administrator." 

![image](https://user-images.githubusercontent.com/59042109/181688745-55c8c54c-ecc8-4170-b1f3-1b6acb5599fd.png)

Please help me resolve this by any means possible.

**Comment by rijkvanzanten** [comment, marked as answer]
V8 is end-of-life and hasn't been updated in years. I would strongly advice against using it at this point.
